features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/sha/sha512_alt.h

```diff
@@ -0,0 +1,107 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2015-2016 Nuvoton
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef MBEDTLS_SHA512_ALT_H
+#define MBEDTLS_SHA512_ALT_H
+
+#if !defined(MBEDTLS_CONFIG_FILE)
```

Comment:
no need to include `config.h` as sha512.h already includes the configuration file before including `sha512_alt.h` file